src/Common/Plugin/NewPluginExtensionModuleRule.php

```diff
@@ -27,7 +27,12 @@ public function apply(AbstractNode $node): void
             return;
         }
 
-        $firstInterface = $node->getInterfaces()[0];
```

Comment:
Would this also work?

$interfaces = $node->getInterfaces();
if (!$interfaces) return
$firstInterface = $interfaces[0];